frontend/src/components/LineChart.vue

```diff
@@ -130,18 +130,36 @@ const { plotStyle, chartData, lineChart } = storeToRefs(chartStore)
 const dataQuality = ref([0, 1, 2, 3])
 const downloading = ref({ csv: false, json: false, chart: false })
 
+// set the initial plot labels. This is overridden in the setParting function
+let xLabel = 'Date'
+let yLabel = `${props.chosenVariable?.name} (${props.chosenVariable?.unit})`
+let title = `${props.data.title}: ${props.chosenVariable?.name} vs Time`
+
 const setParsing = (datasets) => {
   datasets.forEach((dataset) => {
-    dataset.parsing.yAxisKey = props.chosenVariable.abbreviation
+
+    // update the chart based on the selected plot 
+    var plt = props.chosenVariable
```

Comment:
rename `chosenVariable` prop to something like `chosenPlot`

Also you declare `var plt` here on line 142 and you `let plt` on line 203. You can do this because of the scope but it might be less confusing if you use different variable names

Reply:
marking this conversation as unresolved. still need to update the name of the prop in all of the components that use it:
https://github.com/CUAHSI/SWOT-Data-Viewer/blob/cam-406-wse-width-graph/frontend/src/views/DistanceCharts.vue#L35
https://github.com/CUAHSI/SWOT-Data-Viewer/blob/cam-406-wse-width-graph/frontend/src/views/TimeSeriesCharts.vue#L39
https://github.com/CUAHSI/SWOT-Data-Viewer/blob/cam-406-wse-width-graph/frontend/src/views/ChartsDashboardView.vue#L6
https://github.com/CUAHSI/SWOT-Data-Viewer/blob/cam-406-wse-width-graph/frontend/src/components/TheBottomSheet.vue#L31

Reply:
I will push a fix for the last remaining couple `chosenVariable` instances